python/tidy/tests/Cargo.toml

```diff
@@ -4,8 +4,5 @@ version = "0.0.1"
 authors = ["The Servo Project Developers"]
 publish = false
 
-[dependencies]
-test-package = { version = "*" }
-
```

Comment:
I guess this one might be a fake dependency for a unit test expectation.

Reply:
Yes I checked unit tests but not `./mach test-scripts`